runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -325,6 +334,17 @@ public void onChatMessage(ChatMessage event)
 		{
 			createGameTimer(SANFEW);
 		}
+
+		if (event.getMessage().startsWith("You drink some of your") && event.getMessage().contains("overload") && client.getSetting(Varbits.IN_RAID) == 1 && config.showOverload())
+		{
+			createGameTimer(OVERLOAD_RAID);
+		}
+
+		if (event.getMessage().startsWith("You drink some of your") && event.getMessage().contains("prayer enhance") && config.showPrayerEnhance())
```

Comment:
Check config.show<myThing> before checking the message contents, as just checking for true/false is a lot faster than checking if string contains, starts or equals something.